app/model.py

```diff
@@ -15,6 +15,9 @@ def sort_mentors(mentor: Dict) -> Tuple:
             return (
                 mentee["pair_programming"] != mentor["pair_programming"],
                 mentee["job_help"] != mentor["job_help"],
+                mentee["validate_status"] != mentor["validate_status"],
+                mentee["is_active"] != mentor["validate_status"],
```

Comment:
This will sort mentors based on if they have the same availability & status as the mentee. I don't think this is what is needed. Rather edit the way we find potential mentors further down in the code. We need to filter out any mentor that is unavailable for any of three reasons.